### PR TITLE
Busco 1.22 + misc bugfixes

### PR DESCRIPTION
**Pull request by remiolsen**
We have a version of BUSCO installed now that does what it's actually supposed to do. Sadly the format of the log file has changed (@ewels knows all about these kinds of problems), so we're parsing the result by hand.

### CODE REVIEW

File: sciLifeLab_utils/run_assembly_report.py [hunk 1]
```diff
@@ -196,29 +196,38 @@ def collect_results_and_report(validation_sample_dir, assemblies_sample_dir,
...
+        except KeyError:
+            print("No BUSCO data found. Assuming BUSCO was not run.")
+            BUSCO_lineage.append(None)
+            continue
         BUSCO_lineage.append(data_path)
 
         #Find the BUSCO metrics from the result file
-        summary_g = os.path.join(validation_sample_dir, assembler, "BUSCO", "run_*", "short_summary_*")
+        summary_g = os.path.join(validation_sample_dir, assembler, "BUSCO", "run_*", "full_table_*")
         if len(summary_g) > 0:
```
Comment (senthil10) [inline sciLifeLab_utils/run_assembly_report.py]: This line is not part of this PR, but still was it intended to be `if len(glob.glob(summary_a)) >0` ? If not wouldn't it be always `True` as `summary_g` is a string that is defined just above this line 👀

File: sciLifeLab_utils/run_validation.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def main(args):
             sample_YAML.close
 
             # Run the job
-            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.1b1\n"]
+            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.22\nsource $BUSCO_SETUP\n"]
```
Comment (senthil10) [inline sciLifeLab_utils/run_validation.py]: `source $BUSCO_SETUP` is it something intended for `irma` ? Hope it will not affect the jobs while running in `milou-b`

Comment (remiolsen) [inline sciLifeLab_utils/run_validation.py]: Actually. BUSCO is not in the module system on irma yet.

File: sciLifeLab_utils/run_assembly_report.py (hunk 1 above)
Comment (remiolsen) [inline sciLifeLab_utils/run_assembly_report.py]: You are correct! Thanks

**Comment by remiolsen** [comment]
@senthil10 It looks like it's `pip install pandas==0.19.1` that is failing. I have a dumb fix for that though :D

File: sciLifeLab_utils/run_validation.py (hunk 2 above)
Comment (senthil10) [inline sciLifeLab_utils/run_validation.py]: ah ha,. but user should have the environment variable `$BUSCO_SETUP` ready while submitting the jobs I suppose ?

**Comment by remiolsen** [comment]
Yes, please! :wink:
The `$BUSCO_SETUP` envvar is put there by `module load BUSCO`.
